### ISSUE TITLE
Suspicious requirements.txt file

### ISSUE THREAD

**Issue by alimaslax** [madhavmk/Noise2Noise-audio_denoising_without_clean_training_data#6]
What's with the requirements? olefile, webSockets and Certifi?

pysocks, pycparser, etc...

Security and Cryptography: The inclusion of packages like cryptography and argon2-cffi suggests that the program may involve cryptographic operations, such as encryption, decryption, hashing, or secure password handling.

**Comment by madhavmk** [comment]
You are right @alimaslax  . There are some redundant requirements at the moment (this repo originally included a production webserver code to serve denoised audio).
If you have a minimal conda env requirements list, please make a PR, and I can merge it 🙂
